fdbrpc/TLSConnection.actor.cpp

```diff
@@ -374,22 +367,15 @@ static void TLSConnectionLogFunc( const char* event, void* uid_ptr, bool is_erro
 	va_end( ap );
 }
 
-void TLSOptions::init_plugin( std::string const& plugin_path ) {
-	std::string path;
+void TLSOptions::init_plugin() {
+	std::string plugin_path = platform::getDefaultPluginPath("fdb-libressl-plugin");
```

Comment:
There's not much of a point in expanding this out to a filesystem path that will never again exist, right?  It seems easier to just leave this as a fake path of just "fdb-libressl-plugin", so that you have something fixed to match against later on.